Examples/Ising1d/ising1d_sr.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
```

Comment:
please let's use the new headers !

Reply:
what is the new header?

Reply:
Copyright 2021 The NetKet Authors.  All rights reserved.

Reply:
only 1 line?

Reply:
The rest is the same, just the first line changes